pages/vomnibar.coffee

```diff
@@ -96,9 +96,25 @@ class VomnibarUI
       @input.value = @previousInputValue
       @previousInputValue = null
 
-    # Highlight the selected entry, and only the selected entry.
+    # Highlight the selected entry, and only the selected entry, and hide the entries too far away from the
+    # selected
+    [lowest, highest] = @calcRangeForVisibleChildren()
     for i in [0...@completionList.children.length]
-      @completionList.children[i].className = (if i == @selection then "vomnibarSelected" else "")
+      className = (if i == @selection then "vomnibarSelected" else if (lowest <= i < highest) then "" else "vomnibarHided")
```

Comment:
This inline if statement is now complex and it should be unfolded onto multiple lines:

```
      className =
        if i == @selection then "vomnibarSelected"
        else if (lowest <= i < highest) then ""
        else "vomnibarHided"
```